AABB_tree/demo/AABB_tree/Viewer.cpp

```diff
@@ -5,7 +5,7 @@
 #include <CGAL/Qt/CreateOpenGLContext.h>
 
 Viewer::Viewer(QWidget* parent)
-  : QGLViewer(CGAL::Qt::createOpenGLContext(),parent),
+  : QGLViewer(CGAL::Qt::createOpenGLContext(parent),parent),
```

Comment:
@maxGimeno:  the destructor of `QOpenGLFunctions` needs the context. We should check in which order the destructors of `QOpenGLFunctions` and of the context are called.